src/widget/knobeventhandler.h

```diff
@@ -1,22 +1,27 @@
 #pragma once
 
-#include <QMouseEvent>
-#include <QWheelEvent>
 #include <QColor>
 #include <QCursor>
-#include <QPoint>
+#include <QMouseEvent>
 #include <QPixmap>
+#include <QPoint>
+#include <QTimer>
+#include <QWheelEvent>
 
 #include "util/math.h"
 
+// duration the cursor is blanked after a mouse wheel event
+constexpr int wheelEventCursorTimeout = 2800;
```

Comment:
2.8 s feels long is this a debug leftover.

Reply:
whoopsy... yes :roll_eyes: